File: JavaScript/8-kyu/03. Even or Odd/index.test.js
Test.assertEquals(even_or_odd(2),"Even",'even_or_odd(2)')
Test.assertEquals(even_or_odd(1),"Odd", 'even_or_odd(1)')
Test.assertEquals(even_or_odd(-4),"Even",'even_or_odd(-4)')
Test.assertEquals(even_or_odd(-3),"Odd", 'even_or_odd(-3)')
Test.assertEquals(even_or_odd(0), "Even")

Test.assertEquals(even_or_odd(1545452), "Even")
Test.assertEquals(even_or_odd(7) , "Odd")
Test.assertEquals(even_or_odd(78), "Even")
Test.assertEquals(even_or_odd(17), "Odd")
Test.assertEquals(even_or_odd(74156741), "Odd")
Test.assertEquals(even_or_odd(100000), "Even")

var ernd = function(){ return (25+ ~~(Math.random()*25)) * 2 }
var ornd = function(){ return ernd()+1 }

console.log("Test some random values")
for( var r=0, x; r< 6; r++ ){
  if( Math.random()>.5 ){
    x = ernd();
    Test.assertEquals(even_or_odd(x),"Even", "even_or_odd("+x+")")
  }else{
    x = ornd();
    Test.assertEquals(even_or_odd(x),"Odd",  "even_or_odd("+x+")")
  }
}

File: Python/8-kyu/3. Even or Odd/index.test.py
test.assert_equals(even_or_odd(2), "Even", 'even_or_odd(2)')
test.assert_equals(even_or_odd(1), "Odd", 'even_or_odd(1)')
test.expect(even_or_odd(0) == "Even")
test.expect(even_or_odd(1545452) == "Even")
test.expect(even_or_odd(7) == "Odd")
test.expect(even_or_odd(78) == "Even")
test.expect(even_or_odd(17) == "Odd")
test.expect(even_or_odd(74156741) == "Odd")
test.expect(even_or_odd(100000) == "Even")
test.expect(even_or_odd(-123) == "Odd")
test.expect(even_or_odd(-456) == "Even")